apps/site/package.json

```diff
@@ -1,20 +1,6 @@
 {
-  "type": "module",
-  "private": true,
   "name": "@node-core/website",
-  "description": "Nodejs.org Website",
-  "homepage": "https://nodejs.org",
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/nodejs/nodejs.org.git"
-  },
```

Comment:
this key supports the [`npm bugs` command](https://docs.npmjs.com/cli/v9/commands/npm-bugs)

Reply:
These are already on the root package.json